docstore/gcpfirestore/fs.go

```diff
@@ -165,6 +165,10 @@ func OpenCollection(client *vkit.Client, collResourceID, nameField string, opts
 // Providing a function to construct the primary key is useful in two situations: if
 // your desired primary key field is not a string, or if there is more than one field
 // you want to use as a primary key.
+//
+// For the collection to be usable with Query.Delete and Query.Update,
+// nameFunc must work with map[string]interface{} as well as whatever
```

Comment:
I find this comment a bit vague if the user doesn't know about the implementation. Maybe add something like "nameFunc must work with both map and struct types representing the same underlying data structure", an example can be helpful as well, or direct the user to our test struct.

Reply:
Done.